src/users/users.service.ts

```diff
@@ -150,4 +151,26 @@ export class UsersService {
       throw err;
     }
   }
+
+  validateRut(rut: string) {
```

Comment:
This is okay, but you can achieve the same by checking if the value has the desired format by using decorators on your DTO (or creating a custom decorator to validate RUTs)

Reply:
Yes, we are also validating the format, but the thing is that the last digit of a rut is a verification character that depends on the previous digits. For instance, my rut is '16901339-K', but I could enter '16901339-4' and the format is ok but is not a valid rut. Just some extra safety measures 🤓